### PR TITLE
Change COULD to MAY

### PR DESCRIPTION
**Pull request by verdammelt**
Document references RFC2119 as source of definitions of MUST, SHOULD etc. but that document does not contain COULD. Thus changing COULD to MAY.

### CODE REVIEW

**Review by kotp — APPROVED** [review]

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Thank you @verdammelt! I didn't have merge rights before, or I would have merged this in 28 days ago. Your change is absolutely warranted and has been squashed into #60.

**Pull request closed without merging**
